src/ome_zarr_models/_rfc5_transforms/multiscales.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import annotations
+
+from typing import Self
+
+from pydantic import (
+    BaseModel,
+    Field,
+    JsonValue,
+    field_validator,
+    model_validator,
+)
+
+from ome_zarr_models._rfc5_transforms.coordinate_transformations import (
+    CoordinateSystem,
+    CoordinateTransformationType,
+    Scale,
+    Translation,
+)
+from ome_zarr_models.base import BaseAttrs
+from ome_zarr_models.common.validation import check_length
+
+__all__ = ["Dataset"]
+
+
+class Dataset(BaseAttrs):
+    """
+    An element of Multiscale.datasets.
+    """
+
+    path: str
+    # TODO: suggest John not to have a tuple for the coordinate transformations of a
+    #  dataset, since it's either a single scale or a single sequence (with a scale
+    #  and a translation)
+    coordinateTransformations: tuple[CoordinateTransformationType, ...] = Field(
```

Comment:
according to the current text of rfc5, only scale and translation are allowed in here. so a sequence that contains a scale / translation is not permitted. (I don't really understand why this requirement exists)

Reply:
see https://github.com/bogovicj/ngff/blob/52d7924f1522bdf917ea912fc416ae55b3229ebb/0.6-dev/index.bs#L1366-L1368

Reply:
oh also, according to Each dictionary in "datasets" MUST contain the field "coordinateTransformation", whose value is a dictionary that defines a, this field is called `coordinateTransformation`, not `coordinateTransformations`. `coordinateTransformations` is stored in zarr group attrs I think? These names are super confusing!

Reply:
I will change the name from `coordinateTransformations` to `coordinateTransformation` for distinguishing the transformation in `Dataset` from the transformation in `Multiscale`.

Reply:
Actually I checked the examples and the name is actually `coordinateTransformations` and not `coordinateTransformations`. I commented this here https://github.com/ome/ngff/pull/138#discussion_r2087612094.

I would not change the code unless the official examples change. But if turns out not to be a typo, then we can update both the examples and code.